### ISSUE TITLE
[Bug]: Cannot read properties of null (reading 'tagName')

### ISSUE THREAD

**Issue by alecmocatta** [rrweb-io/rrweb#1444]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://www.github.com/rrweb-io/rrweb/issues) for a bug report that matches the one I want to file, without success.

### What package is this bug report for?

rrweb

### Version

2.0.0-alpha.12

### Expected Behavior

No uncaught exceptions.

### Actual Behavior

There is an uncaught exception triggering the window [`error` event](https://developer.mozilla.org/en-US/docs/Web/API/Window/error_event):
```text
TypeError: Cannot read properties of null (reading 'tagName')
```

on this line: https://github.com/rrweb-io/rrweb/blob/3d1877cff83d9a018630674fb6e730050ceef812/packages/rrweb/src/record/mutation.ts#L444

### Steps to Reproduce

In my app press cmd+backspace on any non-empty line in a contentEditable div. Unfortunately my app's not public yet.

### Additional Information

I use rrweb thru [PostHog session replay](https://posthog.com/session-replay) and capture all backtraces from the window error event. Because whatever loads this script (I presume PostHog?) loads it without `crossorigin` set I initially just got a "Script error." error event. It was reproducing locally with the debugger open that I got the aforementioned error and tracked it down to the aforementioned line.

**Comment by metal-messiah** [comment]
To keep this issue alive, many of our customers have also reported seeing this error.  We are using the same version noted above in our instrumentation library.

**Comment by eoghanmurray** [comment]
Fixed in #1445 ?
